tests/lpp_test.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 24 12:12:25 2020
+
+@author: smullally
+"""
+
+
+import exovetter as exo
+import lightkurve as lk
+import astropyfrom astropy import units as u
+from astropy.time import Time
+
+
+def test_one_lpp():
+    #Use Case is get values for one TCE.
+    period = 1.235 * u.d
+    tzero =  24541250.235 * t.jd
+    duration = 5.2 * u.h
+    target_name = "TIC 12345678"
+
+    
+    tce = exo.TCE(period = period, tzero = tzero, duration = duration
+                  target_name = target_name)
+    
+    #Specify the lightcurve to vet
+    mission = "TESS"
+    sector = 14
+    
+    #Generic  function that runs lightkurve and returns a lightkurve object.
+    lc = exo.fetch_TESS_lightcurve(target_name, mission=mission, sector = sector)
+    
+    lpp = exo.Lpp(ddir = "/path/to/file/")
+    
+    result = lpp.apply(tce,lc)
+    
+    assert result['lpp_lpp'] > 0
+    assert result['lpp_normlpp'] > 0
+    assert len(result['lpp_binned']) > 10
+    
+
+def test_run_many_tces():
+    
+    #Each element of vet_list needs to contain the TCE info
+    #and info to create the ligth curve (filename, or lightkurve call)
+    vet_list = exo.load_vet_list('filename_tce.csv')
+    
+    Lpp = exo.lpp('path/to/file')
+    Mod = exo.modsift(model = "trap")
+    Snr = exo.snr()
+    
+    vetter_list = [Lpp, Mod, Snr]
+    
+    results = list()
+    
+    for vet in tce_list:
+        #Details of input are hidden here.
+        tce, lc = exo.load_tce_and_lightcurve(vet)
+        
+        tce_results = dict()
+        
+        tce_results.update(tce)
+        
+        for v in vetter_list:
+            
+            tce_results.update(v.run(tce, lc))
+    
+        results.append(tce_results)
+    
+    assert len(results) == len(vet_list)
```

Comment:
Not sure what this is really testing. It will always pass. When it is not passing, things are going to error out before it gets to this comparison.
```suggestion
    assert len(results) == len(vetter_list)
```